### DISCUSSION TITLE
Files read-only within Helper-Script sub directory

### DISCUSSION THREAD

**Discussion by fbson** [Guilouz/Creality-Helper-Script-Wiki#424] [Issues]
Almost certainly me doing something wrong(!) - but most of the config files within the Helper-Script folder and KAMP within it are (have become - ?) read-only, Is this normal?
Thanks!

**Comment by Guilouz** [comment]
Yes it’s normal, files are symlinked to always have latest files version when updating script

**Reply by fbson** [reply]
Ah, I see. 
So if I want to modify one of the files I can't from Fluidd (for example), I have to SSH into the printer instead to do so, is that correct? Thanks again.

**Comment by Guilouz** [comment]
Yes or you need to duplicate file and include the new one in printer.cfg

**Reply by fbson** [reply]
Ah, that sounds like a convenient alternative - thanks again, and thanks for your work!

**Reply by Guilouz** [reply]
Yes but this files are not supposed to be modified.

**Reply by Mayuna90** [reply]
"Why shouldn't it be changed? I want to turn off the fan where he cleans the nozzle so that it doesn't come on. Is that possible?"

**Reply by dingausmwald** [reply]
Sam here.
It says: "You can change the fan speed, open nozzle-cleaning-fan-control.cfg file located in Helper-Script folder"

but its read only. Pulling my hair out to get to know how to change this damn setting. The docs should be changed. Really confusing.

**Reply by flatirondesigns** [reply]
> Yes but this files are not supposed to be modified.

If we are not supposed to modify the file, how do I permanently change the chamber fan default value the correct way?

**Reply by dingausmwald** [reply]
You read the post you answered to and use your brainz.

My brainz:
duplicate file, f.e. nozzle-cleaning-fan-control-duplicate.cfg
change the line
[include Helper-Script/nozzle-cleaning-fan-control.cfg]
to
[include Helper-Script/nozzle-cleaning-fan-control-duplicate.cfg]
in your printer.cfg

**Reply by flatirondesigns** [reply]
@dingausmwald  Thanks for your response, but you should read the posts first before making insulting responses. 
Unlike you, I wasn't pulling my hair out. I know multiple ways of editing the file. 

Look at Guilouz's comment in the post. He says, the file is not supposed to be modified, which implies there is a more correct way of making changes that don't require modifying the file(which, to me, includes copying). I am asking what those changes are, say through the fluidd's GUI or through the slicer or whatever. 
 
Maybe you should use your brainz... :laughing:
